CodeImprovements/Exercise6.js

```diff
@@ -1,11 +1,5 @@
-function maxNumber(a, b, c) {
-  if (a > b && a > c) {
-    return a;
-  } else {
-    if (b > a && b > c) {
-      return b;
-    } else {
-      return c;
-    }
-  }
-}
+const maxNumber = (a, b, c) => {
+  if (typeof a !== "number" || typeof b !== "number" || typeof c !== "number")
```

Comment:
This is cool😁, but we can utilize the ES6 features and test if all inputs are numbers. 
What if we use the `rest operator` alongside `every() `method to check for types of numbers like this?

```javascript
const maxNumber = (...numbers) => {
    if (!numbers.every((number) => typeof number === 'number')) // then ...
}

```

Reply:
Thank you! This is helpful and I will make sure to incorporate it.